recipes/restinio/all/conanfile.py

```diff
@@ -74,50 +71,51 @@ def requirements(self):
         elif self.options.with_pcre == 2:
             self.requires("pcre2/10.40")
 
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["RESTINIO_INSTALL"] = True
+        tc.variables["RESTINIO_FIND_DEPS"] = False
+        tc.variables["RESTINIO_USE_EXTERNAL_EXPECTED_LITE"] = True
+        tc.variables["RESTINIO_USE_EXTERNAL_OPTIONAL_LITE"] = True
+        tc.variables["RESTINIO_USE_EXTERNAL_STRING_VIEW_LITE"] = True
+        tc.variables["RESTINIO_USE_EXTERNAL_VARIANT_LITE"] = True
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.generate()
+
     def package_id(self):
```

Comment:
could please rearrange methods order? There is a good order here: https://docs.conan.io/en/latest/reference/commands/creator/create.html#methods-execution-order